api/src/routers/meals.js

```diff
@@ -0,0 +1,56 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const meals = express.Router();
+
+meals.get("/", async (req, res) => {
+  const meals = await knex("Meal").orderBy("ID", "asc");
+  res.send(meals);
+});
+
+meals.post("/", async (req, res) => {
+  const body = req.body;
+
+  if (body) {
+    await knex("Meal").insert(body);
+  }
+});
+
+meals.get("/:id", async (req, res) => {
+  const mealId = parseInt(req.params.id, 10);
+  const selectedMeal = await knex("Meal").where("id", mealId);
+  res.send(selectedMeal);
+});
+
+meals.put("/:id", async (req, res) => {
+  const body = req.body;
+  const mealId = parseInt(req.params.id, 10);
+  const selectedMeal = await knex("Meal").where("id", mealId);
+  if (body) {
```

Comment:
probably some sort of parsing and validation would be nice before updateing or inserting anything into db. You never know if the api is getting the accurate data or not :)